py/desitarget/data/targetmask.yaml

```diff
@@ -273,6 +276,9 @@ priorities:
         MWS_MAIN_RED:                 SAME_AS_MWS_MAIN_BLUE
         MWS_MAIN_RED_NORTH:           SAME_AS_MWS_BROAD_NORTH
         MWS_MAIN_RED_SOUTH:           SAME_AS_MWS_BROAD_NORTH
+        MWS_FAINT:                    {UNOBS: 1300, MORE_ZWARN: 2, MORE_ZGOOD: 2, DONE: 2, OBS: 1, DONOTOBSERVE: 0, MORE_MIDZQSO: 0}
```

Comment:
I believe that @djschlegel suggested that these should enter at the lowest priority. Even lower than all of the secondary targets. so, I believe the `UNOBS` priority here needs to be more like `50`.

Reply:
I set UNOBS=50 for MWS-FAINT, but then tests fail on a check that MORE_ZGOOD for BGS (=1000) is always higher than the minimum priority for an unobserved MW target (good test!).

I could update the test to permit this case (i.e. BGS reobservations ahead of MWS-FAINT), or alternatively lower the BGS MORE_ZGOOD. Someone would have to approve the second option.

I guess my biased opinion is that, given the tiny number and patchy distribution of targets we're talking about, unobserved MWS-FAINT should be preferred to BGS reobservations. 

There's also the question of whether BGS reobservations should be higher priority than secondary targets, and whether MORE_ZGOOD=1000 matches whatever the decision is about that. 

Maybe these are questions for @djschlegel?

Reply:
For now, just update the test to except `MWS_FAINT`. I think the BGS group is tending towards _not_ having a high priority for `MORE_ZGOOD` observations. My guess is that `MORE_ZGOOD` will end up as the `DONE` priority for all target classes, as was the case in `sv3`.

Reply:
OK, done.